torchbenchmark/util/framework/transformers/translation/args.py

```diff
@@ -0,0 +1,273 @@
+"""
+Hacked from https://github.com/huggingface/transformers/blob/main/examples/pytorch/translation/run_translation_no_trainer.py 
+It runs HuggingFace transformer models translation on WMT16
+"""
+import argparse
+from transformers import SchedulerType
+
+task_to_keys = {
```

Comment:
I separate the dataset specific args like this. See init py to see how tasks_to_keys is used. Not sure if this is the best way to separate dataset/task specific hf_args and model-specific hf_args.

Reply:
Yep it looks good to me,